### ISSUE TITLE
support for cross domain requests

### ISSUE THREAD

**Issue by narensprinter** [futurepress/react-native-static-server#47]
Is there a way to allow cross domain requests? I have tried to set Access-Control-Allow-Origin but not found how do that. is there any other way?

**Comment by NeliHarbuzava** [comment]
Yeah, it will be good, because now I get CORS errors when trying to make requests to localhost

**Comment by carltian** [comment]
Android, I get CORS errors, too.I want to know how to solve it
